daemon/mgr/cri.go

```diff
@@ -15,6 +16,9 @@ import (
 )
 
 const (
+	pouchRuntimeName = "pouch"
+	kubeAPIVersion   = "0.1.0"
```

Comment:
What does this version mean? kubeAPI has a version?

Reply:
`kubeAPIVersion` means `Version of the kubelet runtime API`.
See [VersionResponse](https://github.com/alibaba/pouch/blob/676e79ab72c7362409bce21d86901ef820b65342/vendor/k8s.io/kubernetes/pkg/kubelet/apis/cri/v1alpha1/runtime/api.pb.go#L268)

Reply:
So I think it is not KubeAPIVersion. KubeRuntimeAPIVersion or RuntimeAPIVersion?